common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from openedx.features.smart_referral.models import SmartReferral
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+
+CONTACT_EMAIL = 'contact_email'
+DAYS_TO_SEND_FOLLOW_UP_EMAIL = 3
+IS_REFERRAL_STEPS_COMPLETED = 'is_referral_step_complete'
+
+
+class Command(BaseCommand):
+    help = """
+    Send follow up referral email to all those contacts
+    """
+
+    def handle(self, *args, **options):
+        emails_list = []
+        three_days_old_date = timezone.now().date() - timedelta(days=DAYS_TO_SEND_FOLLOW_UP_EMAIL)
+
+        three_days_old_referral = SmartReferral.objects.filter(created__date__lte=three_days_old_date).values(
+            CONTACT_EMAIL, IS_REFERRAL_STEPS_COMPLETED)
```

Comment:
Lets filter out contact with `IS_REFERRAL_STEPS_COMPLETED = True` in this query, as well. Once we do that we can remove condition `if not referral[IS_REFERRAL_STEPS_COMPLETED]:`